openghg/standardise/surface/_agage.py

```diff
@@ -84,7 +84,7 @@ def parse_agage(
     species = str(filepath).split(sep="_")[-2]
```

Comment:
Similarly here, if `filepath` is None I believe this would produce an error. Were we able to extract the `species` from the `file_params` details (`file_params = ds.attrs`) instead?